src/evm/onchain/endpoints.rs

```diff
@@ -715,7 +715,7 @@ impl OnChainConfig {
 impl OnChainConfig {
     fn get_pair(&self, token: &str, network: &str, block: &str) -> Vec<PairData> {
         let block_int;
-
+        println!("get pair {} {} {}", token, network, block);
```

Comment:
Is this log necessary? Same question at line 760.

Reply:
When faced with a frozen interface or excessive refreshing,
 I chose to implement screen refreshing to let the user perceive that the program is actively performing tasks.

Reply:
How about writing maxblock (latest synced by graph), also? So user could run consistently.

Reply:
@iVoider This is a temporary solution. In the future, we plan to develop our own graph server to synchronize the latest data for fuzzing purposes.